.github/workflows/iroha2-lts.yml

```diff
@@ -6,49 +6,35 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  REPOSITORY_URL: '${{ github.server_url }}/${{ github.repository }}.git'
 
 jobs:
   deploy:
     runs-on: [self-hosted, Linux]
     container:
       image: 7272721/i2-ci:latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: Swatinem/rust-cache@v1
-      - name: set up buildx
-        uses: docker/setup-buildx-action@v2
-        id: buildx
-        with:
-          install: true
-          driver: docker
-
-      - name: Build iroha2:base image
-        uses: docker/build-push-action@v3
-        with:
-          load: true
-          tags: iroha2:base
-          context: .
-          file: Dockerfile.base
-
       - name: Build iroha2:build docker image
         uses: docker/build-push-action@v3
         with:
           load: true
           tags: iroha2:build
-          context: .
+          context: ${{ env.REPOSITORY_URL }}#iroha2-lts
```

Comment:
Does this result in an extra `git clone`?

Reply:
It seems [like that](https://docs.docker.com/engine/reference/commandline/build/)

Reply:
I think `.` implies `${{ env.REPOSITORY_URL }}#branch`, thanks to `actions/checkout`

Reply:
Taken over by https://github.com/hyperledger/iroha/commit/3796924f02191981ff7cef3ceec4a6343874a95f in #2427